test/connectionManagement.spec.js

```diff
@@ -299,4 +299,80 @@ describe('connection management', () => {
       expect(error.code).toBe(Penpal.ErrorCode.ConnectionDestroyed);
     }
   );
+
+  it('calls onConnectionLost when connection with child is lost', (done) => {
+    const iframe = createAndAddIframe(`${CHILD_SERVER}/default.html`);
+
+    const connection = Penpal.connectToChild({
+      iframe,
+      onConnectionLost: () => {
+        expect(true).toBe(true);
+        done();
+      },
+    });
+
+    connection.promise.then(() => {
+      document.body.removeChild(iframe);
+    });
+  });
```

Comment:
This test is failing for me because it times out after 5 seconds (5 seconds is the Jasmine test timeout period). I believe the test isn't completing in time because removing the iframe doesn't immediately (or even quickly) destroy the connection. The code that destroys the connection when the iframe is removed can be found in [monitorIframeRemoval.ts](https://github.com/Aaronius/penpal/blob/4a1d49f0e0f3a7f198e11772ad82e9d7ea0acff7/src/parent/monitorIframeRemoval.ts). Notice that monitorIframeRemoval.ts only checks to see if the iframe is removed every 60 seconds. If the iframe is not found at that point, then it destroys the connection. 

A somewhat naive or partial approach to address this problem would be either of these options:
* Decrease the interval (currently 60 seconds) that checks whether the iframe is removed.
* Use a [mock clock](https://jasmine.github.io/api/5.5/Clock.html) within the test and advance it by 60 seconds after removing the iframe (see the [_doesn't destroy connection if connection succeeds then timeout passes (connectToChild)_ test](https://github.com/Aaronius/penpal/blob/4a1d49f0e0f3a7f198e11772ad82e9d7ea0acff7/test/connectionManagement.spec.js#L238-L259)  for an example of using a fake clock).

However, there's a bigger question that needs to be addressed as part of https://github.com/Aaronius/penpal/issues/58, which is: What constitutes the connection being lost? I'll expound on this in a comment at the PR level. The solution to this failing test depends on the answer to that question.